metal/loadbalancers/metallb/metallb.go

```diff
@@ -15,10 +15,11 @@ import (
 )
 
 const (
-	hostnameKey      = "kubernetes.io/hostname"
-	serviceNameKey   = "nomatch.metal.equinix.com/service-name"
-	defaultNamespace = "metallb-system"
-	defaultName      = "config"
+	hostnameKey         = "kubernetes.io/hostname"
+	serviceNameKey      = "nomatch.metal.equinix.com/service-name"
+	serviceNameSpaceKey = "nomatch.metal.equinix.com/service-namespace"
```

Comment:
Space is vast.

```suggestion
	serviceNamespaceKey = "nomatch.metal.equinix.com/service-namespace"
```

Reply:
It is, indeed, the final frontier. We can do this.